package.json

```diff
@@ -21,6 +21,7 @@
     "react-i18next": "^11.3.4",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.0",
+    "storybook": "^5.3.17",
```

Comment:
Why? Should already be added

Reply:
should be fixed by rebase

Reply:
and no longer leave `^` before the version number